### PR TITLE
All spelling errors in the comment are fixed

### PR DESCRIPTION
**Pull request by amir734jj**

### CODE REVIEW

File: analyze/aps-bind.c [hunk 1]
```diff
@@ -180,7 +180,7 @@ static SCOPE add_ext_sig(SCOPE old, Declaration tdecl, Signature sig) {
 	      current_type_env = saved;
 	      return new_scope; }
 	  default:
-	    /*! eventually handle renamings */
+	    /*! eventually handle remaining */
```
Comment (boyland) [inline analyze/aps-bind.c:183]: Bad change.

renaming is what I want.  We are not remaining, but renaming things.

File: analyze/aps-fiber.c [hunk 2]
```diff
@@ -755,7 +755,7 @@ void init_field_decls(Declaration module, STATE *s) {
       Use pu = use(def_name(declaration_def(p)));
       Declaration tr = type_renaming(def(intern_symbol("P"),TRUE,FALSE),
 				     type_use(pu));
-      /* shortcut use, I don't think my code handles renamings well */
+      /* shortcut use, I don't think my code handles remaining well */
```
Comment (boyland) [inline analyze/aps-fiber.c:758]: Another error

File: analyze/aps-type.c [hunk 3]
```diff
@@ -1174,7 +1174,7 @@ Type check_pattern_actuals(PatternActuals args, Type ftype, Use type_envs)
   Pattern arg;
 
   switch (Type_KEY(ftype)) {
-    /*! change to handle renamings */
+    /*! change to handle remaining */
```
Comment (boyland) [inline analyze/aps-type.c:1177]: Ditto

**Review by boyland — CHANGES_REQUESTED** [review]
All but three changes looked good to me.

**Comment by amir734jj** [comment]
Sorry, fixed the remaining -> renamings issue

**Review by boyland — APPROVED** [review]
Good.  Thanks for fixing all these typos.

**Pull request merged**